src/views/home/components/header/header.styles.ts

```diff
@@ -0,0 +1,65 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+
+const flexBox = { width: "100%", display: "flex" };
+const btn = (theme: Theme) => ({
+  display: "inline-flex",
+  alignItems: "center",
+  padding: [[theme.spacing(0.75), theme.spacing(1)]],
+  borderRadius: theme.spacing(1),
+  height: theme.spacing(4.5),
+  "& svg": {
+    width: 24,
+  },
+  "& p": {
+    margin: [[0, theme.spacing(1)]],
+    display: "none",
+    [theme.breakpoints.upSm]: {
+      display: "inherit",
+    },
+  },
+  "&:hover": {
+    backgroundColor: theme.palette.grey.light,
+  },
+});
+
+const useHeaderHomeStyles = createUseStyles((theme: Theme) => ({
+  header: {
+    ...flexBox,
+    justifyContent: "space-between",
```

Comment:
Why do we need the `justifyContent: "space-between"` and the `width: "100%"`?

Reply:
True I forgot remove it when I redesigned it

Reply:
But in this case, we don't need `flex: 1`, so instead of the spread I would add `display: "flex"`.